### PR TITLE
Disable gunzip publish

### PR DESCRIPTION
**Pull request by sateeshperi**

### CODE REVIEW

File: tests/bismark_run_qualimap.nf.test.snap [hunk 1]
```diff
@@ -558,7 +556,7 @@
                 "genome_mapping_quality_histogram.png:md5,74776b1e7e8c8876af1cf89d9dbb5f31",
                 "genome_reads_content_per_read_position.png:md5,c6920dc357411880132c207c7d0aa68d",
                 "genome_uniq_read_starts_histogram.png:md5,8f7549a63eee3d3f022699aaa5bb858d",
-                "qualimapReport.html:md5,2c25df556c43190d52453e76de692a13",
+                "qualimapReport.html:md5,bbc015c0d1f3c41a08c82fb7a6839039",
```
Comment (fellen31) [inline tests/bismark_run_qualimap.nf.test.snap:559]: Why is this changed?

Comment (sateeshperi) [reply tests/bismark_run_qualimap.nf.test.snap:559]: @fellen31  not sure. but, it is a html file so must have changed with some formatting done recently.

This should be fine for now and will remove its md5 from snapshot if still variable in future
